wbooks-api/spec/models/book_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+# require 'books_factory'
+
+describe Book do
+  it do
+    is_expected.to validate_presence_of(:genre)
```

Comment:
One thing about this: it's our fault because we still have outdated documentation in the training cards, but we tend to use the should syntax and not the is_expected one
Something like:
```
  it { should validate_presence_of(:name) }
```

Both do the same thing though and don't worry about changing it here, just know that's how you'll usually find it.